src/applications/disability-benefits/686c-674/config/location-schema.js

```diff
@@ -0,0 +1,86 @@
+import { isChapterFieldRequired } from './helpers';
+
+export const locationUISchema = (
+  chapter,
+  outerField,
+  isInsideListLoop,
+  uiTitle,
+  formChapter,
```

Comment:
Could you add some jsdoc comments above this function that summarizes what it does, and what each of these parameters are for?